tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2614,18 +2616,22 @@ func increaseSizeOfPvcAttachedToPod(f *framework.Framework, client clientset.Int
 	pvcConditions := pvclaim.Status.Conditions
 	expectEqual(len(pvcConditions), 0, "pvc should not have conditions")
 
-	var fsSize int64
-	ginkgo.By("Verify filesystem size for mount point /mnt/volume1")
-	fsSize, err = getFSSizeMb(f, pod)
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
-	framework.Logf("File system size after expansion : %v", fsSize)
-	// Filesystem size may be smaller than the size of the block volume
-	// so here we are checking if the new filesystem size is greater than
-	// the original volume size as the filesystem is formatted for the
-	// first time
-	gomega.Expect(fsSize).Should(gomega.BeNumerically(">", originalSizeInMb),
-		fmt.Sprintf("error updating filesystem size for %q. Resulting filesystem size is %d", pvclaim.Name, fsSize))
-	ginkgo.By("File system resize finished successfully")
+	if *pvclaim.Spec.VolumeMode != v1.PersistentVolumeBlock {
+		var fsSize int64
+		ginkgo.By("Verify filesystem size for mount point /mnt/volume1")
+		fsSize, err = getFSSizeMb(f, pod)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		framework.Logf("File system size after expansion : %v", fsSize)
+		// Filesystem size may be smaller than the size of the block volume
+		// so here we are checking if the new filesystem size is greater than
+		// the original volume size as the filesystem is formatted for the
+		// first time
+		gomega.Expect(fsSize).Should(gomega.BeNumerically(">", originalSizeInMb),
+			fmt.Sprintf("error updating filesystem size for %q. Resulting filesystem size is %d", pvclaim.Name, fsSize))
+		ginkgo.By("File system resize finished successfully")
+	} else {
+		ginkgo.By("Volume resize finished successfully")
+	}
 }
```

Comment:
can you perform a filesystem resize manually here on the raw block volume

Reply:
if application has filesystem mounted on top of raw block volume, it is application's responsibility to resize filesystem accordingly.